### DISCUSSION TITLE
The image in excerpt is out of frame

### DISCUSSION THREAD

**Discussion by rockbenben** [vuepress-theme-hope/vuepress-theme-hope#2515] [Q&A] [answered]
If the image in the excerpt is too wide, the image will only be partially displayed.
![image](https://user-images.githubusercontent.com/28252913/206677523-c3e3c22b-c296-4040-bf45-b9f60ce2121e.png)

**Comment by Mister-Hope** [comment, marked as answer]
https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/ded3da93353128cc5f6b0fba88dbb97aa766c2d0
